backend/danswer/configs/chat_configs.py

```diff
@@ -75,6 +75,15 @@
     or "The name of the conversation must be in the same language as the user query."
 )
 
+
+# Agentic search takes significantly more tokens and therefore has much higher cost.
+# This configuration allows users to get a search-only experience with instant results
+# and no involvement from the LLM.
+DISABLE_AGENTIC_SEARCH = (
```

Comment:
Your comment reminded me of the other reason we added this, maybe we should make a note of it as well. Some LLM providers have fairly strict rate limits so this would run out their budget very quickly.

Reply:
Great point! Adding now